src/overview.md

```diff
@@ -1,21 +1,48 @@
-#Augurk Build Tasks
+# Augurk
 
-Install this extension in order to integrate *Augurk* into your build pipeline. At the moment, a single build task is
-provided that publishes your __*.feature__ files to an Augurk instance.
+Install this extension in order to integrate *Augurk* into your build pipeline. At the moment, a single build task is provided that publishes your __*.feature__ files to an Augurk instance.
 
 To get started, use the following steps.
 
-##1. Install Augurk
+## 1. Install Augurk
 Before you can integrate with Augurk you obviously have to install Augurk somewhere. Download the latest release of
-Augurk from our [GitHub](https://github.com/marktaling/Augurk) page and install the WebDeploy package on a server that
-is accessible from Visual Studio Team Services.
+Augurk from our [GitHub](https://github.com/augurk/Augurk/releases) page and install the WebDeploy package on a server that can be accessed by your build agents. If you're using Visual Studio Team Services hosted build agents this means that your Augurk installation must be accessible on the public internet.
```

Comment:
Do we have an example somewhere of how to configure this? Then I'll add that to the documentation.